builds/linux/rhel/8/linux-rhel.pkr.hcl

```diff
@@ -231,6 +232,7 @@ build {
       "--extra-vars", "ansible_username=${var.ansible_username}",
       "--extra-vars", "ansible_key='${var.ansible_key}'",
       "--extra-vars", "enable_cloudinit=${var.vm_guest_os_cloudinit}",
+      "--extra-vars", "{\"rhsm_enabled\": ${var.rhsm_enabled}}",
```

Comment:
Formatting is different than a similar above. Any reason?

Reply:
Ah, good point! Might need to modify the enable_cloudinit in a follow-up PR to do the same.